docs/faq.md

```diff
@@ -501,6 +501,24 @@ If the registry returns `401 Unauthorized` or `"code":"UNAUTHORIZED"`, it is oft
 * Typos in username, password, image names, or registry names.
 * You are using a private registry without HTTPS. See [How can I diagnose problems pulling or pushing from remote registries?](#how-can-i-diagnose-problems-pulling-or-pushing-from-remote-registries).
 
+Note, if Jib was able to retrieve credentials, you should see a log message like these:
```

Comment:
Should we clarify that this is at the `info` level? The `[INFO]` tag is there but maven uses it even for lifecycle messages.

Reply:
I just moved the `[INFO]` prefix. This should create no confusion now. These messages will always be printed (if Jib retrieved credentials) regardless of Maven or Gradle.